src/screens/PaymentDetailsScreen.tsx

```
@@ -5,25 +1,98 @@
+            <input
+              className="input"
+              type="text"
+              placeholder="Amount"
+              value={paymentShares.get(member.name) || ''}
+              onChange={(e) => handleInputChange(member.name, e.target.value)}
+            />
+          </div>
+        </div>
+      ))}
+      <div className="buttons mt-4">
+        <button className="button is-primary" onClick={handleSave}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)